src/server/routes/scanner.js

```diff
@@ -28,6 +28,7 @@ const ruleStatus = {
     warning: 'warning'
 };
 
+const noDocRules = ['optimize-image'];
```

Comment:
Isn't there a way to automatically determine this, so that we don't need to update this list manually?

Reply:
I'm not sure we can do that with the current version of sonarwhal

Reply:
Also this should be only temporary and in a per case base. I'd rather us spend our time in other things 😊

Reply:
> Also this should be only temporary 

Ok.